dpnp/dpnp_iface_searching.py

```diff
@@ -298,35 +296,59 @@ def where(condition, x=None, y=None, /):
 
     For full documentation refer to :obj:`numpy.where`.
 
+    Parameters
+    ----------
+    condition : {dpnp.ndarray, usm_ndarray}
+        Where True, yield `x`, otherwise yield `y`.
+    x, y : {dpnp.ndarray, usm_ndarray, scalar}, optional
+        Values from which to choose. `x`, `y` and `condition` need to be
+        broadcastable to some shape.
+
     Returns
     -------
     y : dpnp.ndarray
         An array with elements from `x` where `condition` is True, and elements
         from `y` elsewhere.
 
-    Limitations
-    -----------
-    Parameter `condition` is supported as either :class:`dpnp.ndarray`
-    or :class:`dpctl.tensor.usm_ndarray`.
-    Parameters `x` and `y` are supported as either scalar, :class:`dpnp.ndarray`
-    or :class:`dpctl.tensor.usm_ndarray`
-    Otherwise the function will be executed sequentially on CPU.
-    Input array data types of `x` and `y` are limited by supported DPNP
-    :ref:`Data types`.
-
     See Also
     --------
-    :obj:`nonzero` : The function that is called when `x` and `y`are omitted.
+    :obj:`dpnp.choose` : Build an array from an index array and a set of
+                         arrays to choose from.
```

Comment:
```suggestion
    :obj:`dpnp.choose` : Construct an array from an index array and a list of
                         arrays to choose from.
```

I think we should follow the main description of functions in Numpy. 
At the moment `dpnp.choose` has a slightly different description but it will change in the future.